code/game/objects/items/weapons/storage/bible.dm

```diff
@@ -1,3 +1,13 @@
+
+//Pretty bible names
+var/global/list/biblenames =		list("Bible", "Koran", "Scrapbook", "Creeper", "White Bible", "Holy Light", "PlainRed", "Tome", "The King in Yellow", "Ithaqua", "Scientology", "the bible melts", "Necronomicon", "Greentext")
+
+//Bible iconstates
+var/global/list/biblestates =		list("bible", "koran", "scrapbook", "creeper", "white", "holylight", "athiest", "tome", "kingyellow", "ithaqua", "scientology", "melted", "necronomicon", "greentext")

```

Comment:
the heck is with this spacing?

Reply:
I think they were trying to line it up on TG so you could see what was what. For us it half works.

Reply:
Then again TG also has relgion on a SS now...just wasn't going there wiht this PR

Reply:
These all need to be made proper global lists with `GLOBAL_LIST_INIT`